### DISCUSSION TITLE
Is it possible to copy the contents of a symlinked config?

### DISCUSSION THREAD

**Discussion by ghost** [twpayne/chezmoi#3302] [Q&A] [answered]
I'm using chezmoi within WSL, where I have a `.wezterm.lua` symlink to the config on my windows drive (`%HOMEPATH%/.wezterm.lua`)

When commiting the config via chezmoi in WSL, it only copies it as a symlink to the windows destination.
Is it possible to set it to grab the file contents itself?

**Comment by Lockszmith-GH** [comment, marked as answer]
There is [this](https://www.chezmoi.io/migrating-from-another-dotfile-manager/#migrate-from-a-dotfile-manager-that-uses-symlinks).
But note, that you're adding the content, and that's what will be restored, it will not overwrite the windows source you're pointing to, it will just preserve the content and next time it will overwrite the symlink.

**Reply by ghost** [reply]
Thank you, this solves my use case.
